bin/github-backup

```diff
@@ -74,6 +74,7 @@ def logging_subprocess(popenargs,
                  child.stderr: stderr_log_level}
 
     def check_io():
+        if sys.platform == 'win32': return
```

Comment:
Mind adding a newline before the return?

Reply:
This should also maybe log a warning at some point.

Reply:
I added a newline and also added a logged warning once per log_subprocess call. maybe there is a better place to log it to ensure it is only noted once?